openfl/component/collaborator/collaborator.py

```diff
@@ -171,6 +174,23 @@ def run(self):
 
                 # Cleaning tensor db
                 self.tensor_db.clean_up(self.db_store_rounds)
+                if self.memleak_check:
+                    # This is the place to check the memory usage of the collaborator
+                    self.logger.info("*****************COLLABORATOR LOGS*******************************")
+                    process = psutil.Process()
+                    process_mem = round(process.memory_info().rss / (1024 ** 2),2)
+                    self.logger.info("Collaborator Round: %s", round_number)
+                    self.logger.info("Collaborator Process Mem: %s", process_mem)
```

Comment:
you actually already do log the memory usage at the process level here ...

Reply:
Yes adding this details in the JSON also.